pubsub/websocket.go

```diff
@@ -213,6 +228,15 @@ type clientHubSpoke struct {
 	c  *hubSpoke
 }
 
+func newClientID() uint64 {
+	var id [8]byte
+	n, err := rand.Read(id[:])
```

Comment:
crypto/rand, but perhaps this should be a monotonically increasing integer counter instead.

Reply:
Just for logging.  I'll switch to a counter.  Although part of the idea is that with the log files that span multiple restarts, grepping for a unique id helps debugging.